_posts/2024-04-16-mai-win-sr2024.md

```diff
@@ -0,0 +1,176 @@
+           caption="The teams and volunteers from SR2024"
+           alt="
+            The competitors and volunteers from the SR2024 competition, standing in and around the arena.
+            At the front the competitors hold a Student Robotics banner, with a selection of robots on the ground in front of them.
+            The volunteers are all wearing the customary blue tee-shirts identifying them as such.
+          "
+%}
+
+[Gymnasium Markt Indersdorf][MAI] took 1st place in Student Robotics 2024 after an exciting final match against [The Ladies' College Guernsey][TLC], [Headington School][HSO] and [Wisbech Grammar School][WGS]. All the teams have worked amazingly hard over the past six months to design, build & program their robots
+
+## The game: Final Frontier
+
+{% include figure.html
+           class="right"
+           src="/images/content/blog/sr2024/arena.png"
+           caption="The 'Final Frontier' arena"
+           alt="
+            A square arena with each of four 'Planet' scoring zones placed along the edges.
+            Those surround an 'Asteroid field' featuring 16 'Asteroids' and in the very center is an 'Egg' on a raised 'Nest'.
+            Robots start in the middle of their home Planet, with a 'Spaceship' (also on their Planet) to their right.
+          "
+%}
+
+This year’s game, *Final Frontier*, plunged teams into an interstellar battle for dominance among four formidable civilizations, all vying for control of the cosmos. Teams had to retrieve Asteroids scattered throughout the Asteroid field and transport them back to their home Planet or, even better, their intergalactic Spaceship currently under construction.
+
+However the stakes are high as within the confines of the asteroid belt lies The Egg -- a cosmic time bomb that, if disturbed, would spell chaos. Teams had to strategise, navigate the treacherous field, and decide whether to safeguard their own civilization or sabotage their rivals. With Asteroids worth points, Spaceships to manage, and The Egg lurking, the universe had never been this competitive.
+
+## Challenges
+
+Following on from last year’s success, this year's competition offered teams a set of Challenges to be completed throughout the competition year. These encouraged teams to start work on their robots early and covered movement, mechanics, and vision. They could be approached in any order, and completing challenges before certain deadlines earned the teams bonus league points.
+
+## Virtual Competition
+
+The first part of our league this year was a virtual competition, broadcast live in March. Across twenty-one matches, teams competed in a simulated version of the main competition, testing their coding skills against one another. Wisbech Grammar School offered the best performance on the day, coming 1st in all four of their matches, though the team from Haberdashers’ Schools remained at the top of the leaderboard due to points they'd earned from the Challenges.
```

Comment:
Suggestion: Might be nice to link to the virtual competition page here, mostly as it contains the livestream.

```suggestion
The first part of our league this year was a [virtual competition]({{ '/events/sr2024/virtual-competition/' | prepend: site.baseurl }}), broadcast live in March. Across twenty-one matches, teams competed in a simulated version of the main competition, testing their coding skills against one another. Wisbech Grammar School offered the best performance on the day, coming 1st in all four of their matches, though the team from Haberdashers’ Schools remained at the top of the leaderboard due to points they'd earned from the Challenges.
```